src/Layout/Tutorial/Tutorial.tsx

```diff
@@ -0,0 +1,49 @@
+import * as React from "react";
+export interface TutorialProps {
+    showTutorial: boolean
+    closeTutorial: () => void
+}
+
+export class Tutorial extends React.Component<TutorialProps, {}> {
+
+    public render() {
+        const showTutorial = this.props.showTutorial ? " is-active" : "" ;
+        const modalClass = "modal" + showTutorial;
+
+        return (
+            <div className={modalClass}>
+            <div className="modal-background" onClick={() => this.props.closeTutorial()}></div>
```

Comment:
Codacy found an issue: [Lambdas are forbidden in JSX attributes due to their rendering performance impact](https://app.codacy.com/gh/Project-Lavinia/Lavinia-client/pullRequest?prid=5947793)